codespell_lib/data/dictionary.txt

```diff
@@ -14821,6 +14847,7 @@ extedns->extends
 extemely->extremely
 exten->extent
 extenal->external
+extendability->extendibility
```

Comment:
I suggest you drop or stash this. 
`extendability` & `extendibility` are both valid as well as invalid depending on which dictionaries you consulted. Validity of both words are inconsistent across many dictionaries.

Both `extendability` & `extendibility` are valid in hunspell, aoo-mozilla-en-dict
Both `extendability` & `extendibility` are invalid in aspell, chrome browser,

`extendability` is valid in hunspell, aoo-mozilla-en-dict
`extendability` is invalid in aspell, chrome browser, ispell

`extendibility` is valid in hunspell, aoo-mozilla-en-dict, ispell
`extendibility` is invalid in aspell, chrome browser

In source code context, in my opinion, majority actually mean `extensibility`.
`extensibility` is also inconsistent in various dictionaries.

https://www.lexico.com/definition/extendability (Both valid, lexico.com has merge with dictionary.com)
https://www.merriam-webster.com/dictionary/extendability  (Both valid)
https://www.dictionary.com/misspelling?term=extendability  (Both valid)
https://wikidiff.com/extensibility/extendibility  (extendibility vs extensibility)

https://www.ldoceonline.com/spellcheck/english/?q=extendability  (Both invalid)
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=extendability  (Both invalid)
https://dictionary.cambridge.org/spellcheck/english/?q=extendibility  (Both invalid)